app/services/feed_service.rb

```diff
@@ -0,0 +1,13 @@
+require 'news-api'
+require 'json'
+
+module FeedService
+  
+  def self.get_articles(category, pagesize)
+    newsapi = News.new(Rails.application.credentials.api[:api_key])
+    top_headlines = newsapi.get_top_headlines(category: category,
+                                              pageSize: pagesize,
+                                              country: 'us',
```

Comment:
Just a thought - can we define a second get_articles method (eg get_SV_articles) that fetches articles for Sweden location? We could then just combine in the controller? might be interesting when we do localisation